src/burn/engine/variable.cpp

```diff
@@ -1897,26 +1897,35 @@ static HRESULT InitializeVariableSystemFolder(
     BOOL f64 = (BOOL)dwpData;
     WCHAR wzSystemFolder[MAX_PATH] = { };
 
-#ifndef _WIN64
```

Comment:
Sounds like @rseanhall, you've come to the same conclusion I did in [my blog post](http://robmensching.com/blog/posts/2017/2/17/lets-code-wix-rooting-payloads-and-fixing-the-system-in-burn/):

> What if this fix breaks backwards compatbility. Then I remind myself this code never returns the right answer. Raymond Chen just wrote about a similar problem. Although our issue isn't a bluescreen or a crash. Hmmm... Nope, I'm still fixing it tonight.

😃